scripts/toaster_configuration

```diff
@@ -13,13 +13,23 @@ MELDIR="$2"
 BITBAKEDIR="$3"
 CMD_LINE="$4"
 
+OE_CORE_NAME="openembedded-core"
+
 BBLAYERS=
 configured_layers () {
    tac "$BUILDDIR/conf/bblayers.conf" | \
    sed -n -e '/^"/,/^BBLAYERS = /{ /^BBLAYERS =/d; /^"/d; p;}' | \
    awk {'print $1'}
 }
 
+get_layer_name () {
+   name=$(grep "BBFILE_COLLECTIONS *+= *\"" ${1}/conf/layer.conf | cut -d '"' -f2)
```

Comment:
This will fail, likely in a very ugly way, if there's more than one of these lines. bitbake doesn't assume only one collection per layer, that's just a convention.